util/__init__.py

```diff
@@ -537,6 +541,191 @@ class LanguageCodes(object):
 zxx|||No linguistic content; Not applicable|pas de contenu linguistique; non applicable
 zza|||Zaza; Dimili; Dimli; Kirdki; Kirmanjki; Zazaki|zaza; dimili; dimli; kirdki; kirmanjki; zazaki"""
 
+    NATIVE_NAMES_RAW_DATA =  [
+        {"code":"ab","name":"Abkhaz","nativeName":"аҧсуа"},
```

Comment:
Any time there's a string like this I would like to use `u''` to turn it into a Unicode string. I think that reduces the risk of encoding errors.